packages/calculator/src/main.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Calculator from '@pie-framework/material-ui-calculator';
+import { withStyles } from 'material-ui/styles';
+import DraggableCalculator from './dragablecalculator';
+import Button from 'material-ui/Button';
+
+class Main extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = { show: false };
+    }
+
+    onToggleShow(){
+        this.setState({
+            show : !this.state.show
+        });
+    }
+
+    render() {
+        const { show } = this.state;
+        const { mode } = this.props.model;
+        return (
+            <div>
+                <Button variant="raised" color="primary" onClick={()=>this.onToggleShow()}>
```

Comment:
`onClick={this.onToggleShow}`, you may need to bind it, you can do that like so: 

```javascript
onToggleShow = () => this.setState({show: !this.state.show})
```